packages/core/lib/console.js

```diff
@@ -27,7 +27,7 @@ const processInput = input => {
 };
 
 class Console extends EventEmitter {
-  constructor(tasks, options) {
+  constructor(allowedCommands, options) {
```

Comment:
`tasks` was previously unused. This should have been caught by `husky`'s `eslint`. I assume this got in because of `--no-verify`. Please be careful when using that commit option.

Reply:
It is a good idea to add a notice.